src/test/java/uk/gov/hmcts/reform/em/stitching/template/DocmosisClientExceptionTest.java

```diff
@@ -45,16 +47,16 @@ private static Response intercept(Interceptor.Chain chain) throws IOException {
                 .build();
     }
 
-    @Test(expected = DocumentTaskProcessingException.class)
-    public void renderTemplate() throws IOException, DocumentTaskProcessingException {
-        client.renderDocmosisTemplate(
-                COVER_PAGE_TEMPLATE_FILE,
-                JsonNodeFactory.instance.objectNode().put("caseNo", "12345"));
-
+    @Test
+    void renderTemplate() {
```

Comment:
Could we make the test name pertinent to what this will test? I.e. throwing a DocumentTaskProcessingException